GW2EIEvtcParser/EncounterLogic/EncounterCategory.cs

```diff
@@ -12,7 +12,8 @@ internal enum FightCategory
         Story,
         OpenWorld,
         UnknownEncounter,
-        Unknown
+        Unknown,
+        Convergence,
```

Comment:
Move that to before the Unknowns, after OpenWorld, Unknown should always be the last defined enum.